spec/features/fill_in_whole_form_spec.rb

```diff
@@ -204,7 +204,7 @@
     expect(form_submission_page).to have_submission_confirmation
     expect(form_submission_page.reference_number).to have_text "142000000100"
     expect(form_submission_page.submission_date).to have_text "13/01/2018 14:00"
-    expect(form_submission_page).to have_download_pdf
+    expect(form_submission_page).to have_invalid_pdf_download
```

Comment:
It looks to me that you have changed this not because of the functionality that you have added, but because the test failed ?
Is it worth sorting this so that the standard data always returns a positive URL so that your javascript can just check it and be happy - not retry after 10 seconds ?

Reply:
Great point, done with purpose in c39c3a9.